docs/2.0/docs/accountfactory/concepts/delegated-repositories.md

```diff
@@ -0,0 +1,38 @@
+# Delegated Repositoires
+
+:::note
+Delegated Repositories are only available to DevOps Foundations Enterprise customers.
+:::
+
+As enterprises scale their usage of IaC across more teams there often arises a need to enforce least-privilege access control to both IaC source code and the CI/CD of that code. Gruntwork's recommendation is to leverage multiple source code repositories to create hard-boundaries that enforce least-privilege access control. The pattern we recommend consists of three parts:
+
+1. A single `infrastructure-live-root` repository, governed by the core platform team. This repository includes IaC for security and shared infrastructure including core-account creation, central logging/auditing and networking controls (e.g. GuardDuty, SecurityHub, Macie, Transit Gateway etc.).
+1. Many _delegated_ repositories, often named using the pattern `infrastructure-live-$TEAM_NAME`, where individual teams can manage their own IaC. These repositories are created by the core platform team using the Account Factory and, via `infrastructure-live-access-control` are granted permissions to manage their own infrastructure.
+git
+Using this pattern central platform teams have the ability to:
+* Centrally manage core compliance and security infrastructure
+* Centrally manage which teams have permission to deploy what kind of infrastructure, allowing central teams to enforce compliance with approved architecture-board approved plans
```

Comment:
```suggestion
* Centrally manage which teams have permission to deploy what kind of infrastructure, allowing their team to enforce compliance with architecture-board approved plans
```

Reply:
I would recommend we just explicitly say "allowing the central platform team to enforce". Using different terms for the team doing the regulation gets hard to follow.